src/openklant/components/klantinteracties/api/tests/test_partijen.py

```diff
@@ -2067,158 +2080,164 @@ def test_invalid_choice_partij_identificator_code_objecttype(self):
                 "codeSoortObjectId": "bsn",
-                "objectId": "296648875",
+                "objectId": "12",
                 "codeRegister": "brp",
             },
         }
+
         response = self.client.post(url, data)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(response.data["code"], "invalid")
         self.assertEqual(response.data["title"], "Invalid input.")
         self.assertEqual(
             response.data["invalid_params"][0]["name"],
-            "partijIdentificatorCodeObjecttype",
+            "partijIdentificator.partijIdentificatorObjectId",
         )
         self.assertEqual(response.data["invalid_params"][0]["code"], "invalid")
         self.assertEqual(
             response.data["invalid_params"][0]["reason"],
-            "voor `codeRegister` brp zijn alleen deze waarden toegestaan: ['natuurlijk_persoon']",
+            "Deze waarde is ongeldig, reden: Waarde moet 9 tekens lang zijn",
         )
 
-    def test_invalid_validation_partij_identificator_code_soort_object_id(self):
-        url = reverse("klantinteracties:partijidentificator-list")
+    def test_invalid_create_empty(self):
+        # all partij_identificator fields required
         partij = PartijFactory.create()
+        list_url = reverse("klantinteracties:partijidentificator-list")
         data = {
             "identificeerdePartij": {"uuid": str(partij.uuid)},
             "anderePartijIdentificator": "anderePartijIdentificator",
-            "partijIdentificator": {
-                "codeObjecttype": "natuurlijk_persoon",
-                "codeSoortObjectId": "kvk_nummer",
-                "objectId": "296648875",
-                "codeRegister": "brp",
-            },
+            "partijIdentificator": {},
         }
 
-        response = self.client.post(url, data)
+        response = self.client.post(list_url, data)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(response.data["code"], "invalid")
         self.assertEqual(response.data["title"], "Invalid input.")
-        self.assertEqual(
-            response.data["invalid_params"][0]["name"],
-            "partijIdentificatorCodeSoortObjectId",
-        )
-        self.assertEqual(response.data["invalid_params"][0]["code"], "invalid")
-        self.assertEqual(
-            response.data["invalid_params"][0]["reason"],
-            "voor `codeObjecttype` natuurlijk_persoon zijn alleen deze waarden toegestaan: ['bsn', 'overig']",
-        )
+        self.assertEqual(len(response.data["invalid_params"]), 4)
 
-    def test_invalid_validation_partij_identificator_object_id(self):
-        url = reverse("klantinteracties:partijidentificator-list")
+    def test_invalid_create_partial(self):
+        # all partij_identificator fields required
         partij = PartijFactory.create()
+        list_url = reverse("klantinteracties:partijidentificator-list")
```

Comment:
I notice that throughout this file this reverse call (and perhaps others as well) is duplicated a lot, can you move it to the class level with `reverse_lazy` instead?